static/js/popovers.js

```diff
@@ -1136,6 +1136,10 @@ export function register_click_handlers() {
         hide_user_profile();
     });
 
+    $("body").on("click", ".stream_list_item", () => {
```

Comment:
This works well but `.stream_list_item` appears to be a very general selector here. I think we should use:
`#user-profile-modal .stream_list_item` here.

Reply:
Yeah! That's probably better. 
Thanks @m-e-l-u-h-a-n!